lib/event_sources.json.example

```diff
@@ -11,7 +11,8 @@
     {
       "ScheduleName": "node-lambda-test-schedule",
       "ScheduleState": "ENABLED",
-      "ScheduleExpression": "rate(1 hour)"
+      "ScheduleExpression": "rate(1 hour)",
+      "ScheduleDescription": "Run node-lambda-test-function once per hour"
```

Comment:
This makes sense and I totally agree. Since there's no section dedicated to ScheduleEvents in the README, I think adding one to document some of the nuances with this feature is warranted. Anyway, if you agree I'd be more than happy to write up the documentation on it.